README.md

```diff
@@ -112,13 +112,15 @@ need to relink it.
 
 ## Using web-ext in NodeJS code
 
-Aside from [using web-ext on the command line][web-ext-user-docs], you may wish to execute `web-ext` in NodeJS code. There is limited support for this. Here are some examples.
+Aside from [using web-ext on the command line][web-ext-user-docs], you may wish to execute `web-ext` in NodeJS code.
+
+`web-ext` npm package exports nodejs native ES modules, and so from a NodeJS CommonJS module they can only be imported using dynamic import.
+
```

Comment:
```suggestion
As of version `7.0.0`, the `web-ext` npm package exports NodeJS native ES modules only. If you are using CommonJS, you will have to use [dynamic imports][].

```

Reply:
Unfortunately, I can make a suggestion at the bottom of this file so you'll have to include:

```
[dynamic imports]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#dynamic_imports
```